tests-legacy/Unit/Core/Cart/CartToOrder/CartToOrderTest.php

```diff
@@ -48,6 +48,9 @@ public function setUp()
     {
         parent::setUp();
 
+        global $kernel;
```

Comment:
why do you need to create the kernel manually here?
in other tests you used `$kernel = self::$kernel;`
maybe this test should extend `SymfonyIntegrationTestCase` so you can do the same?

Reply:
Unfortunately I need the setup of IntegrationTestCase